2.3.1 release appears to have lodash-es issue

I'm getting the following error on compilation:

```
ERROR in ./node_modules/redux-actions/es/createCurriedAction.js
Module not found: Error: Can't resolve 'lodash-es/fp/curryN' in '/Users/tquetano/git/idr-settings-js/node_modules/redux-actions/es'
```
And this started exactly when `2.3.1` was released.

--- a/src/createCurriedAction.js
+++ b/src/createCurriedAction.js
@@ -1,5 +1,5 @@
-import curryN from 'lodash/fp/curryN';
+import curry from 'lodash/curry';
 import createAction from './createAction';
 
 export default (type, payloadCreator) =>
-  curryN(payloadCreator.length, createAction(type, payloadCreator));
+  curry(createAction(type, payloadCreator), payloadCreator.length);

--- a/test/createCurriedAction.test.js
+++ b/test/createCurriedAction.test.js
@@ -1,0 +1,12 @@
+import createCurriedAction from '../src/createCurriedAction';
+
+const type = 'TYPE';
+
+test('returns curried function', () => {
+  const curriedAction = createCurriedAction(type, (a, b) => a + b);
+
+  const a = curriedAction(1);
+  const b = a(2);
+
+  expect(b).toEqual({ payload: 3, type });
+});

